contracts/FundManagement.sol

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity 0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract FundManagement is Ownable {
+    struct Fund {
+        uint256 totalAmount;
+        address asset;
+        uint256 requestWithdrawTime;
+        uint256 amountWithdrawable;
```

Comment:
`amountWithdrawable` appears a bit misleading. It seems this param is more `pendingWithdrawAmount` or something, since it actually starts at 0 and slowly approaches `totalAmount`